packages/typescript/src/plugins/FederatedTypesPlugin.ts

```diff
@@ -288,10 +288,9 @@ export class FederatedTypesPlugin {
   ) {
     try {
       this.logger.log(`Getting types index for remote '${remote}'`);
-      const resp = await axios.get<TypesStatsJson>(
-        `${new URL(origin)}${this.normalizeOptions.typesIndexJsonFileName}`,
-        { timeout: downloadRemoteTypesTimeout },
-      );
+      const indexTypesUrl = new URL(origin);
+      indexTypesUrl.pathname = _path.default.join(indexTypesUrl.pathname, this.normalizeOptions.typesIndexJsonFileName);
```

Comment:
I think this looks right, thanks for contributing! Quick question, is there a reason to use _path.default versus just path since its already imported?

Reply:
apologies, I did this changes as a path within my `node_modules` and didn't realise this. It should be fixed now

Reply:
No problem, mind checking out the failure on formatting when you get some time? Looks like that file is failing